x-pack/test/api_integration/apis/endpoint/resolver.ts

```diff
@@ -15,15 +16,123 @@ const commonHeaders = {
 // eslint-disable-next-line import/no-default-export
 export default function resolverAPIIntegrationTests({ getService }: FtrProviderContext) {
   const supertest = getService('supertest');
+  const esArchiver = getService('esArchiver');
```

Comment:
These tests currently only cover phase0 data, the resolver-generate hasn't been extended to create the phase 1 data yet.